src/main/java/CallableSumCalculator.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class CallableSumCalculator {
+    private static final int LIMIT = 1000;
+
+    public static int getSum(List<Integer> list, int threads) {
```

Comment:
```suggestion
    public int getSum(List<Integer> list, int threads) {
```
no point of making this method static. It will make the code less flexible